packages/itwinui-css/src/table/base.scss

```diff
@@ -371,10 +371,15 @@
 
   // Empty & loading states
   > .iui-table-empty {
+    text-align: center;
     position: sticky;
     inset-inline-start: 50%;
     transform: translateX(-50%);
     padding: var(--iui-size-xl);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    flex-grow: 1;
```

Comment:
required for what

Reply:
`test-align: center` was needed to fix this failing css-workshop test:
![image](https://github.com/iTwin/iTwinUI/assets/45748283/4b408a55-2427-44ac-b107-dd80ae656b82)

And the flex related attributes were required to fix this failing css-workshop test:
![image](https://github.com/iTwin/iTwinUI/assets/45748283/2377683d-4e43-4312-9a44-448425d1eb10)